src/languages/haxe.js

```diff
@@ -89,7 +89,7 @@ export default function(hljs) {
       },
       {
         className: 'type', // instantiation
-        begin: /new */,
+        begin: /\bnew\b/,
```

Comment:
Would this also work:

```
beginKeywords: 'new',
```

Reply:
Depending on what the goal is here a multi-matcher might also be useful... see the docs on `match` for an example... also some of the newer grammers like Ruby or Wren have examples of this in action.  The Haxe grammar looks to be quite old and dusty.

Let me know if you have any questions.

Reply:
I've updated it to use beginKeywords and works good 🤝